besspin/cyberPhys/ui/hacker-kiosk/atem-control-test.js

```diff
@@ -0,0 +1,19 @@
+const { Atem } = require('atem-connection')
+const myAtem = new Atem()
+myAtem.on('info', console.log)
+myAtem.on('error', console.error)
+
+myAtem.connect('192.168.10.240')
```

Comment:
This is an odd address - are you sure this is correct? The local network is 10.88.88.X